### ISSUE TITLE
IndexError on script launch

### ISSUE THREAD

**Issue by aerovolts** [aerovolts/python-scripts#1]
After retrieving several images the script breaks into an IndexError. 

Cause is currently unknown, most likely cause is an empty string of some sort but several attempts at filtering out empty strings failed to work.

**Comment by aerovolts** [comment]
```
Traceback (most recent call last):
  File "redditscrape.py", line 54, in <module>
    main()
  File "redditscrape.py", line 25, in main
    create_folders()
  File "redditscrape.py", line 31, in create_folders
    download_images()
  File "redditscrape.py", line 45, in download_images
    wget.download(post.url, path)
  File "C:\ProgLangs\Python35\lib\site-packages\wget.py", line 527, in download
    filename = detect_filename(url, out, headers)
  File "C:\ProgLangs\Python35\lib\site-packages\wget.py", line 486, in detect_filename
    names["headers"] = filename_from_headers(headers) or ''
  File "C:\ProgLangs\Python35\lib\site-packages\wget.py", line 258, in filename_from_headers
    name = fnames[0].split('=')[1].strip(' \t"')
IndexError: list index out of range
```

**Comment by aerovolts** [comment]
Fixed in commit 2a1b63c66d7721940346ddead8330a6060cdcd2e
